### PR TITLE
DDF-1278: reorganized ui docs.

### PR DESCRIPTION
**Pull request by ricklarsen**
@oscaritoro @clockard @gordocanchola @vinamartin @stustison @djblue

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/464" target="_blank">ReviewNinja

File: catalog/ui/docs/src/main/resources/ManContents.adoc [hunk 1]
```diff
@@ -1,259 +1,70 @@
...
-|normalizationDisabled
-|Boolean
-|Disables relevance and distance normalization.
-|no
-
-|===
-
-==== Standard Search UI
-[cols="1,1,1,4,1" options="header"]
+[cols="1,2,2" options="header"]
```
Comment (oscaritoro) [inline catalog/ui/docs/src/main/resources/ManContents.adoc]: Looks like you are missing the real header row for this table...

Comment (ricklarsen) [inline catalog/ui/docs/src/main/resources/ManContents.adoc]: fixed.

**Comment by oscaritoro** [comment]
Ship it :+1:

File: catalog/ui/docs/src/main/resources/IntContents.adoc [hunk 2]
```diff
@@ -1,38 +1,41 @@
 
-== Overview
-
-The DDF Standard Search UI application allows a user to search for records in the local Catalog (provider) and federated sources. Results of the search are returned in HTML format and are displayed on a globe, providing a visual representation of where the records were found.
+The {branding} Standard Search UI application allows a user to search for records in the local Catalog (provider) and federated sources.
+Results of the search are returned in HTML format and are displayed on a globe, providing a visual representation of where the records were found.
```
Comment (vinamartin) [inline catalog/ui/docs/src/main/resources/IntContents.adoc]: I'd reword this to say "are displayed as points on a globe", but if you want to just take note of it for now and do it later, I'm fine with that. Just a :thought_balloon:

**Comment by vinamartin** [comment]
Ship it!

**Comment by clockard** [comment]
LGTM :+1:

**Comment by gordocanchola** [comment]
:+1:
